### ISSUE TITLE
UI Enhancement: add keyboard-shortcut options for accessing the Permissions-Menu

### ISSUE THREAD

**Issue by umspokbl** [RequestPolicy/requestpolicy#357]
It would be a great enhancement to usability to be able to access the Permissions-Menu via keyboard shortcut (e.g., be able to tap CTRL+SHIFT+<some letter>) and have the Permissions-Menu open and be navigable via arrow keys. Users would be able to tap Enter to make selections and/or Esc to close the Permissions-Menu; users would also be able to tap alphabetical keys that correspond to "underlined-letters in Permissions-Menu descriptions" to directly access that selection.

**Comment by Synzvato** [comment]
I second this. I used to access the options through the right click menu, so that the icon would not always be on-screen. This is no longer possible for me. A keyboard shortcut would be great!

**Comment by dkasak** [comment]
Indeed. I use Pentadactyl for browsing so I have to use the mouse solely for RequestPolicy. This would be a most welcome feature.
